risc0/zkp/Cargo.toml

```diff
@@ -17,6 +17,7 @@ bytemuck = { version = "1.12", features = ["derive"] }
 digest = "0.10"
 fil-rustacuda = { version = "0.1", optional = true }
 hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
+lazy_static = { version = "1.4.0", default-features = false, features = ["spin_no_std"] }
```

Comment:
Needed?

Reply:
Yes, for the Poseidon constants since some of them are calculated.